.travis.yml

```diff
@@ -3,17 +3,14 @@ sudo: required
 language: go
 go:
   - "1.12.x"
+  - "1.13.x"
 
 go_import_path: github.com/containerd/zfs
 
 install:
   - sudo apt-get install -y zfsutils-linux && sudo modprobe zfs
-# currently we don't pin containerd version.
-  - mkdir -p $GOPATH/src/github.com/containerd && cd $GOPATH/src/github.com/containerd && git clone https://github.com/containerd/containerd.git
-# mistifyio/go-zfs (Apr 14, 2019)
-  - mkdir -p $GOPATH/src/github.com/mistifyio && cd $GOPATH/src/github.com/mistifyio && git clone https://github.com/mistifyio/go-zfs.git && cd go-zfs && git checkout f784269be439d704d3dfa1906f45dd848fed2beb
   - cd $GOPATH/src/github.com/containerd/zfs
-  - go get -t ./...
+  - GO111MODULE="on" go mod vendor
   - go get -u github.com/vbatts/git-validation
```

Comment:
after we add vendor check, we cannot call `get get` in module because it will change existing file. will change it in followup.